ci/webpack.yml

```diff
@@ -0,0 +1,29 @@
+name: Webpack
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+    
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Build
+      run: |
+        npm install -g webpack webpack-cli --save-dev
```

Comment:
can't we assume that webpack-cli is already part of package.json? We assumed that gulp and grunt.

Reply:
makes sense, taken care of